kubernetes/structures_pod.go

```diff
@@ -754,6 +757,10 @@ func expandPodSpec(p []interface{}) (*v1.PodSpec, error) {
 		obj.NodeSelector = nodeSelectors
 	}
 
+	if v, ok := in["runtime_class_name"].(string); ok {
```

Comment:
Please add here validation whether `v` is not an empty string. If the attribute is not set it carries an empty string. In this case, the request will be rejected by the Kubernetes API, since the empty string value is not valid for `RuntimeClassName`.

Reply:
Fixed!